data_registry/urls.py

```diff
@@ -13,6 +13,7 @@
     path("search/", views.search, name="search"),
     path("publication/<int:id>", views.detail, name="detail"),
     path("publication/<int:id>/download", views.download_export, name="download"),
+    path("datasets-metadata", views.datasets_metadata, name="datasets_metadata"),
```

Comment:
Let's avoid a new word for publication (registry ocde already misuses "collection").

```suggestion
    path("publications.json", views.publications_api, name="publications_api"),
```